<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="models">
      <item>
       <widget class="QLabel" name="models_label">
        <property name="text">
         <string>models</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="models_lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="models_pushButton">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="height_LineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="width_LineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="5">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="json_label">
        <property name="text">
         <string>json output</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="json_lineEdit">
        <property name="placeholderText">
         <string>write out json output file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="csv_checkBox">
        <property name="text">
         <string>to_csv</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="video_checkBox">
        <property name="text">
         <string>video output</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="4">
     <widget class="QGroupBox" name="hand_groupBox">
      <property name="title">
       <string>Hand</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="handJoint_horizontalLayout">
         <item>
          <widget class="QLabel" name="handJoint_label">
           <property name="text">
            <string>hand joint</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="hand_comboBox">
           <item>
            <property name="text">
             <string>Hand21</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Type</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QRadioButton" name="media_radioButton">
         <property name="text">
          <string>Image/Video</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="camera_radioButton">
         <property name="text">
          <string>Camera</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="camera_groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="camidx_label">
            <property name="text">
             <string>index</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="camidx_spinBox">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="media_groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="media_lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="media_pushButton">
            <property name="text">
             <string>open</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QGroupBox" name="body_groupBox">
      <property name="title">
       <string>Body</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="bodyJoint_horizontalLayout">
         <item>
          <widget class="QLabel" name="bodyJoint_label">
           <property name="text">
            <string>body joint</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="body_comboBox">
           <item>
            <property name="text">
             <string>j12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>j18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>j20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>j23</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>j25</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QGroupBox" name="head_groupBox">
      <property name="title">
       <string>Head</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="headJoint_horizontalLayout">
         <item>
          <widget class="QLabel" name="headJoint_label">
           <property name="text">
            <string>head joint</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="head_comboBox">
           <item>
            <property name="text">
             <string>Face20</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QCheckBox" name="render3d_checkBox">
        <property name="text">
         <string>3D</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ik_checkBox">
        <property name="text">
         <string>use ik</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>434</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionexecute"/>
   <addaction name="actioncli"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionexecute">
   <property name="text">
    <string>execute</string>
   </property>
  </action>
  <action name="actioncli">
   <property name="text">
    <string>use CLI</string>
   </property>
   <property name="toolTip">
    <string>cli</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
